### DISCUSSION TITLE
Strand + or - ?

### DISCUSSION THREAD

**Discussion by YDD99** [Kinggerm/GetOrganelle#258] [Q&A]
why after use evaluate_assembly_using_mapping.py, then annotate modified.fasta is different original fasta. gene direction different from the original assembled gene direction after the'evaluate_assembly_using_mapping.py'is used. BUT its not happend all the time ,just happend Occasionally occurring .As shown in the following figure.

**Comment by Kinggerm** [comment]
The use of `modified.fasta` is designed 1) for accommodating bowtie2 which cannot do circular mapping for generating circular-mapping results, and 2) for the consistency of the evaluation, given biologically the same sequence with different forms, e.g. different start point of a circular sequence. For the second purpose, the sequence will be normalized, in many ways.
